core/import_data/import_business_plans.py

```diff
@@ -0,0 +1,327 @@
+        year_end=bp_data["year_end"],
+        country=bp_data["country"],
+        defaults=bp_data,
+    )
+
+    return business_plan
+
+
+def add_business_plan_values(business_plan, row, columns_dict):
+    """
+    Add business plan values
+
+    @param business_plan: BusinessPlan object
+    @param row: row data
+    @param columns_dict: dict of columns
+    """
+
+    values = []
+    for year, columns in columns_dict.items():
+        value_data = {}
+        # set values
+        for value_type in ["usd", "odp", "mt"]:
+            column_name = columns.get(value_type)
+            if not column_name or not row[column_name]:
+                continue
+            value_data[f"value_{value_type}"] = get_decimal_from_excel_string(
+                row[column_name]
+            )
+
+        if not value_data:
+            continue
+        value_data.update(
+            {
+                "business_plan_id": business_plan.id,
+                "year": year,
+            }
+        )
+        values.append(BPValue(**value_data))
+
+    BPValue.objects.bulk_create(values)
+
+
+def add_chemicals(business_plan, row, index_row):
+    """
+    Add chemicals to business plan
+
+    @param business_plan: BusinessPlan object
+    @param row: row data
+    @param index_row: row index
+    """
+    if not row["Chemical detail"]:
+        return
+
+    chemicals = row["Chemical detail"].split("/")
+    substances = []
+    blends = []
+    for chemical_name in chemicals:
+        chemical, ch_type = get_chemical_by_name_or_components(chemical_name)
+        if not chemical:
+            logger.warning(
+                f"[row: {index_row}]: "
+                f"Missing chemical: {chemical_name} ({chemicals})"
+            )
+            continue
+        if ch_type == "substance":
+            substances.append(chemical)
+        elif ch_type == "blend":
+            blends.append(chemical)
+
+    business_plan.substances.add(*substances)
+    business_plan.blends.add(*blends)
+
+
+def parse_file(file_path, file_name):
+    """
+    Parse file and create business plans
+
+    @param file_path: file path
+    @param file_name: file name
+    """
+    df = pd.read_excel(file_path, dtype=str).replace({np.nan: None})
+
+    # get prerequisites data
+    year_start, year_end = re.search(YEAR_REGEX, file_name).groups()
+    year_start, year_end = int(year_start), int(year_end)
+    df = rename_columns(df)
+    year_columns_dict = get_values_columns(df, year_start, year_end)
+
+    for index_row, row in df.iterrows():
+        bp = create_business_plan(row, index_row, year_start, year_end)
+        if not bp:
+            continue
+        add_business_plan_values(bp, row, year_columns_dict)
+        add_chemicals(bp, row, index_row)
+
+
+@transaction.atomic
+def import_business_plans():
+    dir_path = settings.IMPORT_DATA_DIR / "business_plans"
+    directory = os.fsencode(dir_path)
+
+    for file in os.listdir(directory):
```

Comment:
Since we are working with pathlib here we can probably just do:

```suggestion
    for file in dir_path.glob("*.xlsx"):
```

Although this works either way, feel free to leave as is.